nucypher/config/constants.py

```diff
@@ -77,4 +77,4 @@
 
 
 # Probationary period (see #2353, #2584)
```

Comment:
We could get rid of this comment altogether, but if keeping - 

```suggestion
# Probationary period (see #2779)
```

Reply:
✔️